sklearn/metrics/_ranking.py

```diff
@@ -759,7 +766,9 @@ def precision_recall_curve(y_true, probas_pred, *, pos_label=None,
         pos_label should be explicitly given.
 
     probas_pred : ndarray of shape (n_samples,)
-        Estimated probabilities or output of a decision function.
+        Target scores, can either be probability estimates of the positive
```

Comment:
```suggestion
        This can either be probability estimates of the positive
```

Reply:
Here again, I aligned the definition of the parameter `probas_pred/y_score` across all the analogue functions: average_precision_score, det_curve, roc_curve, label_ranking_average_precision_score, coverage_error, label_ranking_loss, dcg_score, ndcg_score, top_k_accuracy_score.